app/scripts/modules/netflix/tableau/application/appTableau.controller.ts

```diff
@@ -0,0 +1,30 @@
+import {IComponentController, ISCEService, module} from 'angular';
+
+import {AUTHENTICATION_SERVICE, AuthenticationService} from 'core/authentication/authentication.service';
+import {NetflixSettings} from '../../netflix.settings';
+import {Application} from 'core/application/application.model';
+
+require('../tableau.less');
+
+class ApplicationTableauController implements IComponentController {
+
+  public srcUrl: string;
+
+  static get $inject(): string[] {
+    return ['$sce', 'app', 'authenticationService'];
+  }
+
+  constructor(private $sce: ISCEService,
+              private app: Application,
+              private authenticationService: AuthenticationService) {
+
+    const user: string[] = this.authenticationService.getAuthenticatedUser().name.split('@');
+    const url =
+      NetflixSettings.tableau.appSourceUrl.replace('${app}', this.app.name).replace('${user}', user[0]);
```

Comment:
So this is the only thing that's _really_ different.  The original file did `[foo] = 'bar@bat.com'.split('@');` which results in `foo === 'bar'` but I don't know what it's called to look it up.  Happy to change `user[0]` back as it definitely feels weird.